### PR TITLE
fix bug - Lazy load BOPIS outlet components

### PR DESCRIPTION
**Pull request by surya-prakash-singh1**
fix bug - Lazy load BOPIS outlet components

### CODE REVIEW

**Comment by WeizhengSap** [comment]
Is `StoreFinderModule` required in `PdpPickupOptionsContainerModule`?

**Review by WeizhengSap — CHANGES_REQUESTED** [review]
Is `StoreFinderModule` required in `PdpPickupOptionsContainerModule`?
e2e tests failed

File: feature-libs/storefinder/root/facade/store-finder.facade.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        'getStoresLoaded',
+        'getFindStoresEntities',
+        'getViewAllStoresLoading',
+        'getViewAllStoresEntities',
+        'findStoresAction',
+        'viewAllStores',
+        'viewStoreById',
+        'callFindStoresAction',
+        'getStoreLatitude',
+        'getStoreLongitude',
```
Comment (WeizhengSap) [inline feature-libs/storefinder/root/facade/store-finder.facade.ts:38]: missing `getFindStoreEntityById`

File: feature-libs/storefinder/root/model/store-finder.model.ts [hunk 2]
```diff
@@ -4,6 +4,12 @@
...
  */
 
+import { PointOfService } from '@spartacus/core';
+
 export enum StoreFinderOutlets {
   PREFERRED_STORE = 'cx-pick-up-in-store-make-my-store',
 }
+
+export interface StoreEntities {
+  pointOfServices?: Array<PointOfService>;
```
Comment (WeizhengSap) [inline feature-libs/storefinder/root/model/store-finder.model.ts]: `StoreEntities` in `storefinder/core` should be removed.

Comment (surya-prakash-singh1) [reply feature-libs/storefinder/root/model/store-finder.model.ts]: Done

**Review by WeizhengSap — APPROVED** [review]

**Pull request merged**